src/R/Support/Impl/Help/IntelliSenseRSession.cs

```diff
@@ -37,27 +46,32 @@ public sealed class IntelliSenseRSession : IIntellisenseRSession {
         public IRSession Session { get; private set; }
 
         public void Dispose() {
+            if (_workflow?.RSession != null) {
```

Comment:
`RSession` is set in ctor, so even if session wasn't started, it still shouldn't be null.